behave/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.7
+
+RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
+RUN sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'
+RUN apt-get update -y && \
+    apt-get install -y google-chrome-stable chromedriver
```

Comment:
You might want to freeze `chromedriver` (as well as selenium and behave below).

Reply:
I would do that if I would be able to install `chromedriver` as pip package. Don't actually know how to do that (and if that's possible) from a perspective od debian system-based package.

Reply:
You can use https://chromedriver.storage.googleapis.com/, there are all of the versions.